safe/definitions/exposure_classifications.py

```diff
@@ -29,6 +30,21 @@
 # to generate the value_mapping. Please do not remove it.
 # See https://github.com/kartoza/osm-reporter/wiki how to use this key.
 
+do_not_report_class = {
+    'key': DO_NOT_REPORT,
+    'name': tr('Do Not Report'),
+    'description': tr(
+        'Land use types that will be excluded in the analysis.'),
```

Comment:
Ah, now you make its own variable. Why not making it generic? (not using `land use`) and to add it to roads/buildings?